tutor/templates/kustomization.yml

```diff
@@ -59,8 +59,12 @@ configMapGenerator:
         app.kubernetes.io/name: redis
 {{ patch("kustomization-configmapgenerator") }}
 
+{%- if patch("k8s-override") or patch("kustomization-patches-strategic-merge") %}
```

Comment:
@keithgg I may don't get it, but why are these ifs are needed?

Reply:
@gabor-boros these changes are part of https://github.com/overhangio/tutor/pull/705. If you don't have those patches by default, you'll get an error.